tests/components/dlna_dmr/test_media_player.py

```diff
@@ -31,13 +31,10 @@
 from homeassistant.components.dlna_dmr.data import EventListenAddr
 from homeassistant.components.dlna_dmr.media_player import DlnaDmrEntity
 from homeassistant.components.media_player import (
-    ATTR_TO_PROPERTY,
-    DOMAIN as MP_DOMAIN,
     MediaPlayerEntityFeature,
     MediaPlayerState,
     MediaType,
     RepeatMode,
-    const as mp_const,
```

Comment:
Importing `const` triggers the pylint warning, so moving the import directly to the component root.